spec/request/rating_info_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'rating info', type: :request do
+  include SchemaMatchers
+
+  let(:user) { build(:disabilities_compensation_user) }
+
+  before do
+    sign_in_as(user)
+  end
+
+  describe 'GET /v0/disability_compensation_form/find_rating_info_pid' do
+    context 'with a valid 200 evss response' do
+      it 'matches the rating info schema' do
```

Comment:
This isn't matching a schema -- is a line missing?

Reply:
It doesn't match a schema because for the factory user, this value is null.